openapi.yaml

```diff
@@ -2738,116 +2700,134 @@ components:
           minimum: 0
           maximum: 1        
                                
+    limit-audit-trail: 
```

Comment:
is `limit-provenance` too precious or too imprecise? I think "where did the limit come from" is the exact purpose of this info object, so `provenance` fits, and I think `audit` implies once use of this info.

We're using `provenance` in other places and the description, so I take that to mean your conversations haven't been tripped up with other customers by using that term, which would be a good reason to use something more accessible.

We can change this later, it shouldn't hold up approval.

Reply:
Agreed, provenance is more consistent an accurate.  I think I was biased as I started thinking about this from the perspective of DYN calling it the "audit" data.  I changed it.